diff -Naur python-telegram-bot-8.1.1/telegram/utils/request.py python-telegram-bot-8.1.1_b/telegram/utils/request.py
--- python-telegram-bot-8.1.1/telegram/utils/request.py	2017-09-25 20:58:53.000000000 +0200
+++ python-telegram-bot-8.1.1_b/telegram/utils/request.py	2017-10-16 23:58:25.559457415 +0200
@@ -31,10 +31,10 @@
 import certifi
 
 try:
-    import telegram.vendor.ptb_urllib3.urllib3 as urllib3
-    import telegram.vendor.ptb_urllib3.urllib3.contrib.appengine as appengine
-    from telegram.vendor.ptb_urllib3.urllib3.connection import HTTPConnection
-    from telegram.vendor.ptb_urllib3.urllib3.util.timeout import Timeout
+    import urllib3 as urllib3
+    import urllib3.contrib.appengine as appengine
+    from urllib3.connection import HTTPConnection
+    from urllib3.util.timeout import Timeout
 except ImportError:  # pragma: no cover
     warnings.warn("python-telegram-bot wasn't properly installed. Please refer to README.rst on "
                   "how to properly install.")
@@ -92,7 +92,7 @@
         kwargs = dict(
             maxsize=con_pool_size,
             cert_reqs='CERT_REQUIRED',
-            ca_certs=certifi.where(),
+            ca_certs="/etc/ssl/certs/ca-certificates.crt",
             socket_options=sockopts,
             timeout=urllib3.Timeout(
                 connect=self._connect_timeout, read=read_timeout, total=None))
